src/modal/modal-stack.ts

```diff
@@ -46,8 +46,9 @@ export class NgbModalStack {
   }
 
   open(moduleCFR: ComponentFactoryResolver, contentInjector: Injector, content: any, options): NgbModalRef {
-    const containerEl =
-        isDefined(options.container) ? this._document.querySelector(options.container) : this._document.body;
+    const containerEl = isDomObject(options.container) ? options.container : isDefined(options.container) ?
```

Comment:
so the `sth instanceof HTMLElement` woudl return falsy for anything but HTMLElement, allowing to simplify these lines